courses/cunix2/libft/tester/tests/ft_memccpy_test.cpp

```diff
@@ -20,10 +20,8 @@ int main(void)
 	memset(dest, 'A', 100);
 	/* 1 */ check(ft_memccpy(dest, "coucou", 0, 0) == 0 && dest[0] == 'A'); showLeaks();
 	/* 2 */ check(ft_memccpy(dest, "coucou", 0, 1) == 0 && dest[0] == 'c' && dest[1] == 'A'); showLeaks();
-	dest[0] = 'A';
 	/* 3 */ check(ft_memccpy(dest, "coucou", 'c', 10) == dest + 1 && dest[0] == 'c' && dest[1] == 'A'); showLeaks();
-	dest[0] = 'A';
-	/* 4 */ check(ft_memccpy(dest, "B", 0, 10) == dest + 2 && dest[0] == 'B' && dest[1] == 0); showLeaks();
+	/* 4 */ check(ft_memccpy(dest, "B", 0, 10) == NULL && dest[0] == 'B' && dest[1] == 0); showLeaks();
```

Comment:
Why NULL? Original function fails with this test, please revert changes for this line and I will merge